Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/ArtifactHelpers/FieldsHelperIntegrationTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using kCura.Relativity.Client.DTOs;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using Relativity.API;
+using Relativity.Test.Helpers.ArtifactHelpers;
+using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
+using Relativity.Test.Helpers.ServiceFactory.Extentions;
+using Relativity.Test.Helpers.SharedTestHelpers;
+
+namespace Relativity.Test.Helpers.NUnit.Integration.ArtifactHelpers
+{
+	[TestFixture]
+	public class FieldsHelperIntegrationTests
+	{
+		private IFieldsHelper SuT;
+		private IHttpRequestHelper _httpRequestHelper;
+		private IHelper _testHelper;
+		private IServicesMgr _servicesManager;
+		private IRSAPIClient _client;
```

Comment:
can we rename it to _rsapiClient so when reading code it can easily point to what client it is.